--- configure.in.orig	2008-06-07 13:08:37.000000000 +0200
+++ configure.in	2008-06-07 13:09:48.000000000 +0200
@@ -132,13 +132,25 @@
 dnl -------------------------------------------------------------------
 dnl Check for gtkspell >= 2.0 and use it if found
 dnl -------------------------------------------------------------------
-AC_MSG_CHECKING([for a gtkspell >= $GTKSPELL_OPTIONAL])
-PKG_CHECK_EXISTS([gtkspell-2.0 >= $GTKSPELL_OPTIONAL], [
-        PKG_CHECK_MODULES(GTKSPELL, [gtkspell-2.0 >= $GTKSPELL_OPTIONAL])
-        AC_DEFINE(HAVE_GTKSPELL, [true], [A usable GtkSpell library was found])
-],[
-        echo "not found - spellchecking disabled."
-])
+have_gtkspell=no
+AC_ARG_WITH(gtkspell,
+            AC_HELP_STRING([--with-gtkspell],
+                           [Build with spell support]),,
+            with_gtkspell=auto)
+
+if test x"$with_gtkspell" != "xno"
+then
+        PKG_CHECK_MODULES([GTKSPELL], [
+                gtkspell-2.0 >= $GTKSPELL_OPTIONAL
+        ], have_gtkspell=yes, have_gtkspell=no)
+
+        if test x"$have_gtkspell" = "xyes"
+        then
+                AC_DEFINE([HAVE_GTKSPELL], 1, [A usable GtkSpell library was found])
+        fi
+
+fi
+
 AC_SUBST(GTKSPELL_CFLAGS)
 AC_SUBST(GTKSPELL_LIBS)
 
